### PR TITLE
Iss7

### PR DESCRIPTION
**Pull request by dneise**
trying to solve issue #7

### CODE REVIEW

**Comment by maxnoe** [comment]
Should this really be in plotting? Maybe an own submodule like `pixelmapping` would make more sense? THis can then be used in plotting.

**Comment by dneise** [comment]
good question. `get_pixel_coords` is in plotting at the moment. But I have no strong opinion.

**Comment by maxnoe** [comment]
Yeah, i think because `get_pixel_coords` was not thought for outside use. But if this is a feature for users, it should not be in plotting.

File: fact/plotting/mapping.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import pkg_resources as res
+import pandas as pd
+import numpy as np
+
+__bias_to_trigger_patch_map = None
+
+def bias_to_trigger_patch_map():
+    global __bias_to_trigger_patch_map
```
Comment (maxnoe) [inline fact/plotting/mapping.py]: Is this kind of caching? Better use `from functools import lru_cache` and `@lru_cache`

**Comment by dneise** [comment]
maybe we can merge this now.
But I have no idea if everything still works. We have no units tests, and I forgot what I should test manually.

File: fact/pixels.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    else:
+        pixel_x = pixel_x_chid
+        pixel_y = pixel_y_chid
+
+    return pixel_x, pixel_y
+
+
+@lru_cache(maxsize=1)
+def bias_to_trigger_patch_map():
+    mapfile = res.resource_filename("fact", "resources/FACTmap111030.txt")
```
Comment (maxnoe) [inline fact/pixels.py]: You do not have to load the file again, it is in the namespace of this submodule

Comment (dneise) [inline fact/pixels.py]: does this speak against merging?

**Comment by dneise** [comment]
please don't merge yet

I need the credentials  ... I would like to include them into this package now...

**Comment by maxnoe** [comment]
Ok

**Comment by dneise** [comment]
okay done

File: fact/pixels.py (hunk 2 above)
Comment (dneise) [inline fact/pixels.py]: Ah  now I understand. Thanks ,... I'll use that

**Pull request merged**